Source/CuckooGeneratorFramework/Tokenizer.swift

```diff
@@ -205,4 +206,30 @@ public struct Tokenizer {
             return Attributes.none
         }
     }
+    
+    private func tokenizeImports(otherTokens: [Token]) -> [Token] {
+        let rangesToIgnore: [Range<Int>] = otherTokens.flatMap { token in
+            switch token {
+            case let container as ContainerToken:
+                return container.range
+            case let extensionToken as ExtensionDeclaration:
+                return extensionToken.range
+            default:
+                return nil
+            }
+        }
+        do {
+            let regex = try NSRegularExpression(pattern: "(?:\\b|;)import(?:\\s|(?:\\/\\/.*\\n)|(?:\\/\\*.*\\*\\/))+([^\\s;\\/]+)", options: [])
+            let results = regex.matchesInString(source, options: [], range: NSMakeRange(0, source.characters.count))
+            return results.filter { result in
+                    rangesToIgnore.filter { $0 ~= result.range.location }.count == 0
+                }.map {
+                    let range = $0.range.toRange()!
```

Comment:
I can change it to `$0.range.location..<($0.range.location + $0.range.length)`, which should produce the same result.